src/components/QACertTestSummaryHeaderInfo/QAImportModalSelect/QAImportModalSelect.js

```diff
@@ -6,7 +6,6 @@ const QAImportModalSelect = ({ setImportTypeSelection }) => {
     { key: "select", name: "Select Data Type to Import" },
     { key: "file", name: "Import from File" },
     { key: "historical", name: "Import Historical Data" },
-    { key: "mats", name: "MATS Data" },
```

Comment:
This removes the option from the _Cert Events, Extensions, & Exemptions_ screen **AND** the _Test Data_ screen. Per the acceptance criteria, it should only be removed from _Cert Events, Extensions, & Exemptions_. This component is used by the `QACertEventHeaderInfo` and `QACertTestSummaryHeaderInfo` components: a prop should be added that can be used to conditionally include this option, for example:
```javascript
    ...(qaType === "testSummary" ? { key: "mats", name: "MATS Data" } : {})
```

Reply:
Sorry, there was an error in my suggestion, but what you committed is correct when combined with the change in `DropdownSelection.js`.

Reply:
Just to note, what I _meant_ to put in my suggestion is:
```javascript
...(qaType === "testSummary" ? [{ key: "mats", name: "MATS Data" }] : [])
```
This would alleviate the need for the empty object check in `DropdownSelection.js` (because of how the spread operator works).

Reply:
@maxdiebold-erg  If I do the ...(qaType === "testSummary" ? [{ key: "mats", name: "MATS Data" }] : []) code change there is still an empty option element. We don't want that. I think we should keep the if statement I added to skip any empty objects. 
![image](https://github.com/user-attachments/assets/1e7bd9cc-812d-43fd-9065-8f9d7ff84ca2)

Reply:
Added ...(qaType === "testSummary" ? [{ key: "mats", name: "MATS Data" }] : []) change